.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -90,4 +90,6 @@ will share this change with the people who need to know about it._
 
 ## [optional] What gif best describes this PR or how it makes you feel?
 
-![alt_text](gif_link)
+<!-- ![gif-name](link)  or -->
+<!-- <img src="gif-link" width="120" height="auto"/> -->
```

Comment:
Quick suggestion: I would prefer that we fully revert this change and leave this as it was prior to changes (as shown below)--thoughts, @Ridhwana?
```suggestion
![alt_text](gif_link)
```